packages/portalnetwork/src/client/client.ts

```diff
@@ -368,18 +374,10 @@ export class PortalNetwork extends (EventEmitter as { new(): PortalNetworkEventE
 
     // private handleContent = async (srcId: string, message: Italk)
 
-    private handleUTPStreamRequest = async (srcId: string, msgId: bigint, packetBuffer: Buffer) => {
-
+    private handleUTP = async (srcId: string, msgId: bigint, packetBuffer: Buffer) => {
+        await this.client.sendTalkResp(srcId, msgId, new Uint8Array())
```

Comment:
What is this line doing?  As far as I know, the current state of the uTP spec is to not ever use the `TALKRESP` message so it's not obvious to me what this is doing by sending a blank response.  Is it just to avoid the error messages that happen when the `TALKREQ` handler in discv5 times out?